roles/st2repos/tasks/debian.yml

```diff
@@ -0,0 +1,23 @@
+---
+- name: Install prereqs (Debian)
+  become: yes
+  apt:
+    name: "{{ item }}"
+    state: present
+    update_cache: yes
+  with_items:
+    - debian-archive-keyring
+    - apt-transport-https
+
+- name: Add keys to keyring
+  become: yes
+  apt_key:
```

Comment:
I think it would be more solid if we check `id` (identifier of key) here as well.
See example with Mongo: https://github.com/StackStorm/ansible-st2/blob/3785d849135599de1f667e692576d3501af67fb5/roles/mongodb/tasks/mongodb_apt.yaml#L6